packages/node/src/methods/app-instance/install-virtual/operation.ts

```diff
@@ -24,6 +23,15 @@ export async function installVirtual(
 
   let updatedStateChannelsMap: Map<string, StateChannel>;
 
+  if (
+    proposal.initiatorDepositTokenAddress !==
```

Comment:
Instead of doing `proposal.<property>` multiple times in this function body, you can destructure it like:

```typescript
const {
  initiatorDepositTokenAddress,
  responderDepositTokenAddress,
  initialState,
  ...
}
```